cmd/apiserver/main.go

```diff
@@ -13,8 +17,32 @@ import (
 const port = ":8080"
 
 func main() {
+	err := godotenv.Load()
+	if err != nil {
+		logging.Logger().Fatal("Error loading .env file")
+	}
+
+	useFile, _ := os.LookupEnv("USE_FILE")
+	var bookRep bookDomain.BookRepository
+	if useFile != "true" {
+		err = repository.ApplyMigrations("file://migrations", "postgres://go-book:go-book@localhost:5432/go-book?sslmode=disable")
+		if err != nil {
+			logging.Logger().Error("Error to apply migrations:", err)
+		}
 
-	bookRep := repository.NewFileBookRepository("books")
+		p, err := repository.NewPgxpool("postgres://go-book:go-book@localhost:5432/go-book?sslmode=disable")
+		if err != nil {
+			logging.Logger().Error("Error to create PostgreSQL connection pool:", err)
+			return
+		}
+
+		defer p.Close()
```

Comment:
use context for waiting until pgxpool will be closed or deadline will be exceeded (Close will block until all connections are returned to pool and closed https://pkg.go.dev/github.com/jackc/pgx/v5/pgxpool#Pool.Close)